well-being/README.md

```diff
@@ -0,0 +1,16 @@
+Mental health can affect any of us at any point in our life’s, mental health doesn't discriminate, and Cyber Connect Scotland believe we shouldn’t either. We’ve compiled a list of some great resources that are available to you to use should you or any one you know ever be looking for some additional help. According to time to change “Statistically, 1 in 4 of us will fight a mental health problem in any given year. That’s why our work is so important. No one should have to fear being treated differently because of a mental health problem.”
+
+In addition to the resources below on the Cyber Connect Scotland slack workspace there is also #well-being channel if you wish to ask questions or chat.
+
+ 
+SAMH 
+https://www.samh.org.uk
+SAMH is the Scottish Association for Mental Health. Operating in communities to provide a range of mental health support and services.
+
+Headspace
+https://www.headspace.com/
+An app providing hundreds of themed meditations on everything from stress and sleep to focus and anxiety
+ 
+Nightline
+https://www.headspace.com/	
```

Comment:
Should this be the NL website? https://www.nightline.ac.uk/want-to-talk/

Reply:
yes - yes it should